ENVI_ImageBandDifference.md

```diff
@@ -50,43 +52,43 @@ Mandatory (optional) settings are listed as Required = True (Required = False).
-input_raster1|True|None| Requires two rasters to detect change, (input_raster1, input_raster2)|Specify a single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
-input_raster1_metadata|False|None| [More on ENVI input port](insert links here)|Provide a dictionary of attributes for overriding the raster metadata. -- Value Type: DICTIONARY
-input_raster1_band_grouping|False|None| [More on ENVI input port](insert links here)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
-input_raster1_filename|False|None| Requires two rasters to detect change, (input_raster1, input_raster2)|Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
-input_raster2|True|None| |Specify a second single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
-input_raster2_metadata|False|None| [More on ENVI input port](insert links here)|Provide a dictionary of attributes for overriding the raster metadata. -- Value Type: DICTIONARY
-input_raster2_band_grouping|False|None| [More on ENVI input port](insert links here)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
-input_raster2_filename|False|None| |Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
+input_raster1|True|None| A valid S3 folder containing image files. |Specify a single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
+input_raster1_format|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the format of the image, for example: landsat-8. -- Value Type: STRING
+input_raster1_band_grouping|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the name of the band grouping to be used in the task, ie - panchromatic. -- Value Type: STRING
+input_raster1_filename|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster) |Provide the explicit relative raster filename that ENVI will open. This overrides any file lookup in the task runner. -- Value Type: STRING
+input_raster2|True|None|  A valid S3 folder containing image files.  |Specify a second single-band raster on which to perform an image difference of input band. -- Value Type: ENVIRASTER
+input_raster2_format|False|None| [More on ENVIRASTER input port](https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster)|Provide the format of the image, for example: landsat-8. -- Value Type: DICTIONARY
```

Comment:
@creeder23 the metadata port is currently being removed and the new docs should reflect this. You can see the changes here https://github.com/TDG-Platform/docs/blob/master/ENVI_Task_Runner_Inputs.md#enviraster